connectors/migrations/20241218_backfill_notion_unkown.ts

```diff
@@ -0,0 +1,35 @@
+import { makeScript } from "scripts/helpers";
+
+import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
+import { concurrentExecutor } from "@connectors/lib/async_utils";
+import { upsertDataSourceFolder } from "@connectors/lib/data_sources";
+import { ConnectorResource } from "@connectors/resources/connector_resource";
+
+makeScript({}, async ({ execute }, logger) => {
+  const connectors = await ConnectorResource.listByType("notion", {});
+
+  await concurrentExecutor(
+    connectors,
+    async (connector) => {
+      const folderId = `notion-unknown`;
+      if (execute) {
+        await upsertDataSourceFolder({
+          dataSourceConfig: dataSourceConfigFromConnector(connector),
+          folderId,
+          parents: [folderId],
+          parentId: null,
+          title: "Orphaned Resources",
+          mimeType: "application/vnd.dust.notion.page",
```

Comment:
Should we use a specific mimetype ? I think we likely don't want to let people "select all" orphaned resources (makes no sense to do this IMO), so we may be able to leverage a specific mime type for that ?

Reply:
Agreed, "notion-unknown" node is not a notion page, so it indeed should not have this mimetype
application/vnd.dust.notion.unknown-folder should do

Reply:
done, do I have to add this mimetype in core too?

Reply:
nope